src/overlays/actors/ovl_En_Horse_Zelda/z_en_horse_zelda.c

```diff
@@ -100,34 +101,34 @@ static InitChainEntry sInitChain[] = {
 };
 
 static EnHorseZeldaActionFunc sActionFuncs[] = {
-    func_80A6DCCC,
-    func_80A6DDFC,
+    EnHorseZelda_Stop,
+    EnHorseZelda_Gallop,
 };
 
-void func_80A6D8D0(unknownStruct* data, s32 index, Vec3f* vec) {
-    vec->x = data[index].unk_0.x;
-    vec->y = data[index].unk_0.y;
-    vec->z = data[index].unk_0.z;
+void EnHorseZelda_GetFieldPosition(HorsePosSpeed* data, s32 index, Vec3f* vec) {
```

Comment:
```suggestion
void EnHorseZelda_GetFieldPosition(HorsePosSpeed* data, s32 index, Vec3f* fieldPos) {
```
Helps clarify what it actually is used for, rather than just a generic vec. Could possibly also rename data to something like `fieldData` but not 100% sure on that.